instructions.cpp

```diff
@@ -1671,6 +1693,7 @@ bool interpret(Module *m) {
 
                 // conversion operations
             case 0xa7 ... 0xbb:
```

Comment:
Should the range be `0xa7 ... 0xbf` to also include the reinterpretation instructions, such as `f32.reinterpret_i32`?
These are implemented in the `i_instr_conversion` function after all.

If this is correct the two cases can become one single `case 0xa7 ... 0xc4` clause.

Reply:
This is correct. The core spec tests in conversions.wast throws the expected exception:

```
unrecognized opcode 0xbd
```